Omulosi/iReporter
# #162686758 Add tests for all API endpoints

#### What does this PR do?
Adds tests to all API endpoints
#### Description of Task to be completed?
Add unit test to each endpoint
#### How should this be manually tested?
clone the repo and checkout this branch. Run the tests using pytest as so: `pytest app/tests/v2`.
All tests should pass
#### What are the relevant pivotal tracker stories?
#162686758
